core/src/display_object/morph_shape.rs

```diff
@@ -39,8 +44,9 @@ impl<'gc> MorphShape<'gc> {
         self.0.read().ratio
     }
 
-    pub fn set_ratio(&mut self, gc_context: MutationContext<'gc, '_>, ratio: u16) {
-        self.0.write(gc_context).ratio = ratio;
+    pub fn set_ratio(&mut self, context: &mut UpdateContext<'_, 'gc, '_>, ratio: u16) {
```

Comment:
What's the motivation for this change? `set_ratio` doesn't really need `context`, but just `gc_context`.

Reply:
Good catch, this was accidental (originally I had lazily initialized whenever the ratio changed, but switched to doing it on render/getting bounds).